src/components/layouts/LoadingSpinner.js

```diff
@@ -1,9 +1,27 @@
-import React from 'react'
+import React from "react";
+import { BallBeat } from "react-pure-loaders";
 
 export default function LoadingSpinner() {
-    return (
-        <div>
-            Loading...
-        </div>
-    )
+  return (
+    <div
+      style={{
```

Comment:
It will be great if you use makeStyles and useStyles

Reply:
Can you elaborate it please, how and where I should use it exactly?